doc/howtos/booting-and-installation.rst

```diff
@@ -147,6 +147,140 @@ Flashing an Intel Edison requires use of a breakout board and two micro-USB cabl
 #. Plug in the second micro-USB cable to the J16 connector as instructed by the running flashall script.
 #. Wait for all the images to flash. You will see the progress on the flasher.
 #. Once flashing is done, the image will automatically boot up and auto-login as ``root``, no password is required.
+   
+BeagleBone Black
+================
+
+BeagleBone Black is booted from a microSD card with MBR (Master Boot Record) and not GPT (GUID Partition Table) partitions.
+Most freshly unpackaged microSD cards come with MBR partitions, but previously used ones might not.  (We have
+instructions below to properly initialize the microSD card.)
+
+You'll probably need an adapter to use the microSD card on your host computer. If you use a microSD-to-SD adapter, 
+it will likely show up as ``/dev/mmcblk0`` when plugged into your host computer. 
+You can verify the device name assigned by using ``dmesg`` or the
+``lsblk`` command to look for the device name for the microSD card (check for a device with the size you're expecting). 
+
+In our setup steps below, we're using an 8GB microSD card in an SD adapter that's showing up as ``/dev/mmcblk0``  
+(numbers and device name maybe different for your device and system).
+
+.. comment:   steps derived from http://www.armhf.com/boards/beaglebone-black/bbb-sd-install/
+.. _BeagleBone build 405 images: https://download.ostroproject.org/builds/ostro-os/2016-03-11_05-44-23-build-405/images/beaglebone/
+.. _Ostro Project download server: http://download.ostroproject.org
+
+1. We'll start by gathering files we'll put on the microSD card.  Aim your browser to the
+   `Ostro Project download server`_ (if you're not doing your own build).
+   The ``releases`` folder contains milestone builds of the Ostro OS, while the
+   ``builds`` folder has non-milestone builds.  For this example, we're using the `BeagleBone build 405 images`_ folder.
+
+   Download these four files to your host computer::
+
+      MLO 
+      ostro-image-dev-beaglebone-*.rootfs.tar.bz2
```

Comment:
Can you add a note indicating that there are other variants available and all should work. The fact we use -dev is just an example.